mgnify_pipelines_toolkit/analysis/assembly/summarise_sanntis_bgcs.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright 2024-2025 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import fileinput
+import logging
+import os.path
+
+import pandas as pd
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s - %(levelname)s: %(message)s"
+)
+
+"""
+Script parses sanntis GFF output and adds descriptions of annotated MIBiGs classes.
+
+Descriptions were provided by Santiago Fragoso in doc with links to papers
+https://docs.google.com/document/d/1IIGokcEpX_yLjg1H-w4_VAbekGRl6LgjSvafym3KMz0/edit?tab=t.0
+Those were copied to TSV file constants/bgc_descriptions/sanntis.txt
+"""
```

Comment:
This text should be in the script help, it's useful to have this printed when using `--help`

Reply:
And that document is good stuff for the documentation of MGnify itself.